scripts/test_usb_scan.py

```diff
@@ -109,10 +106,7 @@ def test_log(m):
 class TestUsbScan(unittest.TestCase):
 
     def setUp(self):
-        try:
```

Comment:
Remove the redundant try...except block, which didn't handle any exception just raise again.